test/routes.test.ts

```diff
@@ -10,39 +10,145 @@ describe('Gyms API', () => {
     const response = await request(app).get('/api/v1/gyms/top').expect(200);
     expect(response.body.topReviewGyms.length).toBe(3);
```

Comment:
I would prefer to have tests, For every route in separate file

Reply:
I will do it